src/main/java/calculator/Application.java

```diff
@@ -1,7 +1,17 @@
 package calculator;
 
+import calculator.domain.Calculator;
+import calculator.domain.Converter;
+import calculator.domain.Number;
+import camp.nextstep.edu.missionutils.Console;
+
+import java.util.List;
+
 public class Application {
     public static void main(String[] args) {
         // TODO: 프로그램 구현
+        Calculator calculator =new Calculator();
```

Comment:
해결된 주석은 삭제하기 & 컨벤션 지키기 😉 
```suggestion
        Calculator calculator = new Calculator();
```

Reply:
다음부터 주의하겠습니다!